test_runner/regress/test_tenant_conf.py

```diff
@@ -336,10 +336,15 @@ def test_live_reconfig_get_evictions_low_residence_duration_metric_threshold(
 ):
     neon_env_builder.enable_pageserver_remote_storage(RemoteStorageKind.LOCAL_FS)
 
-    env = neon_env_builder.init_start()
+    env = neon_env_builder.init_start(
+        initial_tenant_conf={
+            # disable compaction so that it will not download the layer for repartitioning
+            "compaction_period": "0s"
+        }
+    )
     assert isinstance(env.pageserver_remote_storage, LocalFsStorage)
 
-    (tenant_id, timeline_id) = env.neon_cli.create_tenant()
+    (tenant_id, timeline_id) = env.initial_tenant, env.initial_timeline
     ps_http = env.pageserver.http_client()
```

Comment:
This is a drive-by fix unrelated to the main change, right?

Would be nice to extract this into a preliminary PR, but, not insisting.


```suggestion
    env = neon_env_builder.init_start(
        initial_tenant_conf={
            # disable compaction so that it will not download the layer for repartitioning
            # (this test does assertions based on manual on-demand download & eviction)
            "compaction_period": "0s"
        }
    )
    assert isinstance(env.pageserver_remote_storage, LocalFsStorage)

    (tenant_id, timeline_id) = env.initial_tenant, env.initial_timeline
    ps_http = env.pageserver.http_client()
```

Reply:
It was a test failure: https://github.com/neondatabase/neon/pull/5108#issuecomment-1823036202

Does your suggestion do something else than the comment? IMHO not stricly needed, it's quite clear what the test does.

Yeah, can split.

Reply:
Split to #5926